.pre-commit-config.yaml

```diff
@@ -0,0 +1,58 @@
+---
+default_stages: [commit]
+
+default_language_version:
+  python: python3
+
+exclude: "examples/.*$"
+
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.4.0
+    hooks:
+      - id: check-ast
+      - id: check-builtin-literals
+      - id: check-byte-order-marker
+      - id: check-executables-have-shebangs
+        # Need to define stages explicitly since `default_stages` was not being respected
+        stages: [commit]
+      - id: check-merge-conflict
+      - id: debug-statements
+      - id: forbid-new-submodules
+      - id: no-commit-to-branch
+      - id: trailing-whitespace
```

Comment:
can you add the new line hook that i added to talos?

Reply:
Yeah sure, i can add in a follow up